workspaces/lightspeed/packages/app/e2e-tests/utils/sidebar.ts

```diff
@@ -49,3 +49,11 @@ export async function assertDrawerState(page: Page, state: 'open' | 'closed') {
     await expectations[state](locator);
   }
 }
+
+export async function sidePanelAssertions(page: Page) {
```

Comment:
I'd suggest a less nebulous function name. Something like `assertConversationExists`, since that's what it seems to be doing

Reply:
either works for me